discover/Home/core-features/sophon-name-service.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Usernames - Sophon Name Service (SNS)"
+description: "How to easily claim your onchain username forever"
+---
+
+Your username is your digital identity in the Sophon universe - a simple ID that connects all your activities.
+
+Instead of sharing a complex wallet address, people can find you, send you things, and recognize your contributions using a memorable name like [username].[soph.id](http://soph.id).
```

Comment:
mintilify creaated a link here. Needs to be removed.

Maybe also cool saying this username also works for your home profile url